Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1252,6 +1207,11 @@ namespace Babylon
         delete texture;
     }
 
+    void NativeEngine::UpdateRawTexture(const Napi::CallbackInfo& info)
+    {
+        // todo
```

Comment:
Why do we need this? If we do need it, it should implemented?

Reply:
I've started updating nativeengine.ts. Removing for now.